corehq/apps/geospatial/routing_solvers/pulp.py

```diff
@@ -3,10 +3,34 @@
 import pulp
 import copy
 
+from dataclasses import dataclass
 from .mapbox_utils import validate_routing_request
 from corehq.apps.geospatial.routing_solvers.base import DisbursementAlgorithmSolverInterface
 
 
+@dataclass
+class Parameters:
+    min_cases_per_user: int
+    max_cases_per_user: int
+    max_case_distance: int
+    max_case_travel_time_seconds: int
+    user_count: int
+    case_count: int
+
+    def __init__(self, config, user_count, case_count):
+        self.user_count = user_count
+        self.case_count = case_count
+
+        default_max_cases = int(case_count / user_count) + 1 if user_count else 0
+        self.max_cases_per_user = config.max_cases_per_user or default_max_cases
+
+        self.min_cases_per_user = config.min_cases_per_user or 0
+        self.max_case_distance = config.max_case_distance
+
+        max_travel_time_secs = config.max_case_travel_time * 60 if config.max_case_travel_time else None
+        self.max_case_travel_time_seconds = max_travel_time_secs if config.supports_travel_mode else None
```

Comment:
nit:
max_travel_time_secs is redundant if travel mode isn't supported, so calculating that would not be worth it.

So, may be this could be better written as
```
self.max_case_travel_time_seconds = None

if config.supports.travel_mode and config.max_case_travel_time:
    self.max_case_travel_time_seconds = config.max_case_travel_time * 60

```